lib/galaxy/tools/parser/yaml.py

```diff
@@ -162,6 +162,12 @@ def parse_tests_to_dict(self):
     def parse_profile(self):
         return self.root_dict.get("profile", "16.04")
 
+    def parse_python_template_version(self):
+        python_template_version = self.root_dict.get("python_template_version", None)
+        if python_template_version is not None:
+            python_template_version = float(python_template_version)
```

Comment:
```suggestion
            python_template_version = packaging.version.parse(python_template_version)
```

You'd need to also add `import packaging.version` at the top.